### ISSUE TITLE
vcard:Title

### ISSUE THREAD

**Issue by tawahle** [openrif/vivo-isf-ontology#747]
When creating a preferred label (via the same-named faux property) some Vcard classes are used, which can be found neither in the latest nor in the older Vcard specification. These classes are: vcard:Organizational and vcard:Title.

**Comment by tawahle** [comment]
Another sub-classes of vcard:Organizational as vcard:Logo, vcard:Related, vcard:OrganizationName are also not specified as classes in the Vcard specification.

**Comment by mjaved495** [comment]
@tawahle  There are also a number of other entities that are obsolete from other ontologies. We will handle these obsolete entities once we have a single vivo ontology file.

**Comment by mconlon17** [comment]
Yes.  There should be tickets like this one to make changes to the ontology.  That way, we have a record of what we found and what we did.

**Comment by tawahle** [comment]
@mjaved495 I'm not sure, if these Vcard entities are obsolet. However, in the latest version of the Vcard ontology they are specified in a completely different way. I think, all the Vcard entities used in VIVO should be checked, since there is also a number of properties which has been changed in the specification, like e.g.   http://www.w3.org/2006/vcard/ns#honorificPrefix in VIVO and http://www.w3.org/2006/vcard/ns#honorific-prefix in Vcard specification.

**Comment by tawahle** [comment]
@mconlon17 Thank you, Mike!

**Comment by mconlon17** [comment]
What is being proposed by this issue?  That we add properties?  Remove them?

**Comment by tawahle** [comment]
I've detected the above mentioned disparities and proposed changes on the Vcard in VIVO in line with the current version of Vcard spek were proposed.  The vivo.owl and the Vcard part in it have actually been aligned with the current Vcard spec during the first VIVO ontology sprint. The question is what are the further steps to implement these changes ... But it was supposed to be a task for a next sprint.
